pkg/resources/scim_integration.go

```diff
@@ -17,40 +20,49 @@ var scimIntegrationSchema = map[string]*schema.Schema{
 		Type:        schema.TypeString,
 		Required:    true,
 		ForceNew:    true,
-		Description: "Specifies the name of the SCIM integration. This name follows the rules for Object Identifiers. The name should be unique among security integrations in your account.",
+		Description: "String that specifies the identifier (i.e. name) for the integration; must be unique in your account.",
+	},
+	"enabled": {
+		Type:        schema.TypeBool,
+		Required:    true,
+		Description: "Specify whether the security integration is enabled.",
 	},
 	"scim_client": {
 		Type:        schema.TypeString,
 		Required:    true,
+		ForceNew:    true,
 		Description: "Specifies the client type for the scim integration",
-		ValidateFunc: validation.StringInSlice([]string{
-			"OKTA", "AZURE", "GENERIC",
-		}, true),
-		DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
-			normalize := func(s string) string {
-				return strings.ToUpper(strings.ReplaceAll(s, "-", ""))
-			}
-			return normalize(old) == normalize(new)
-		},
+		ValidateFunc: validation.StringInSlice(sdk.AsStringList([]sdk.ScimSecurityIntegrationScimClientOption{
+			sdk.ScimSecurityIntegrationScimClientOkta, sdk.ScimSecurityIntegrationScimClientAzure, sdk.ScimSecurityIntegrationScimClientGeneric,
+		}), true),
+		DiffSuppressFunc: ignoreTrimSpaceSuppressFunc,
 	},
-	"provisioner_role": {
+	"run_as_role": {
```

Comment:
is a role also an account object identifier?

Reply:
Yes, but it's an enum of 3 roles below, that's why I decided not to validate as AccountObjectIdentifier.